scripts/quicklink-to-project.sh

```diff
@@ -43,4 +43,4 @@ fi
 
 echo ""
 echo "Beginning integration..."
-yarn concurrently --kill-others --names "$CURRENT_NAME,$PROJECT_NAME" "yarn watch" "cd $PROJECT && ./scripts/quicklink.sh @artsy/reaction"
\ No newline at end of file
+yarn concurrently --kill-others --names "$CURRENT_NAME,$CURRENT_NAME,$PROJECT_NAME" "yarn watch" "node scripts/publish-on-change.js" "cd $PROJECT && ./scripts/quicklink.sh @artsy/reaction"
```

Comment:
I still wanted the `yalc publish` step to be viewed as coming from reaction, so I just added the same name twice for concurrently.

Reply:
👍